### PR TITLE
fix(css/minifier): do not remove custom variables with different case

### PR DESCRIPTION
**Pull request by alexander-akait**
**Description:**

Bug fix - `--color: red;` and `--Color: blue` are different variables

**BREAKING CHANGE:**

No

**Related issue (if exists):**

No

### CODE REVIEW

File: crates/swc_css_minifier/src/compressor/rules.rs [hunk 1]
```diff
@@ -58,10 +58,25 @@ impl Compressor {
...
+    fn is_same_declaration_name(&self, left: &Declaration, right: &Declaration) -> bool {
+        match (&left.name, &right.name) {
+            (
+                DeclarationName::Ident(Ident {
+                    value: left_value, ..
+                }),
+                DeclarationName::Ident(Ident {
+                    value: right_value, ..
+                }),
+            ) => left_value.to_ascii_lowercase() == right_value.to_ascii_lowercase(),
```
Comment (kdy1) [inline crates/swc_css_minifier/src/compressor/rules.rs:70]: Let's use `eq_ignore_ascii_case`

Comment (alexander-akait) [reply crates/swc_css_minifier/src/compressor/rules.rs:70]: @kdy1 I will fix, anyway, declaration name will be normalize too in future, just do it step by step :smile:

**Comment by alexander-akait** [comment]
@kdy1 Fixed :+1:

**Review by kdy1 — DISMISSED** [review]
swc-bump:
 - swc_css_minifier

**Review by kdy1 — APPROVED** [review]

**Pull request merged**